lib/data_update_scripts/20210503175027_move_summary_and_location_to_profile.rb

```diff
@@ -0,0 +1,29 @@
+module DataUpdateScripts
+  class MoveSummaryAndLocationToProfile
+    def run
+      # do we also need to delete the old data?
+      ActiveRecord::Base.connection.execute(
+        <<~SQL.squish,
+          BEGIN;
+          WITH extracted_data AS (
+            SELECT user_id,
+                   data->'summary' as summary,
+                   data->'location' as location,
+                   data->'website_url' as website_url
+            FROM profiles
+            WHERE data->'summary' IS NOT NULL
```

Comment:
I wonder if this check is actually more expensive than just inserting the `null` values unconditionally.

Reply:
Yep, I tried to run in with Blazer and it went over the timeout and froze the tab. I think it's because `data->{something}` has no index, and there are 600k `profiles`. 

Option B: given that the the new columns are nullable, and given there are 600k rows, you could also just do it in Rails using batches:

```ruby
Profile.select(:id, :user_id, "data->'summary' as data_summary").find_each do |profile|
  profile.update_columns(summary: profile.data_summary)
end
```